sacro-app/package.json

```
@@ -1,31 +1,36 @@
 {
-  "name": "sacro-app",
-  "productName": "sacro-app",
-  "version": "0.1.0",
-  "description": "SACRO Outputs Viewer",
-  "main": "src/index.js",
+  "name": "sacro",
+  "version": "1.0.0",
+  "main": "main.js",
   "scripts": {
-    "start": "electron-forge start",
-    "package": "electron-forge package",
-    "make": "electron-forge make",
-    "publish": "electron-forge publish",
-    "lint": "echo \"No linting configured\""
+    "run": "electron . --no-sandbox",
```

```suggestion
    "dev": "electron . --no-sandbox",
```

Could we use `dev` or `start` as the keyword here to match other Node.js projects?